python/IECoreHoudini/TestCase.py

```diff
@@ -37,10 +37,48 @@
 
 ## A class to help implement unit tests for Houdini functionality. It
 # implements setUp() to create a new houdini scene to perform the test in.
+
 class TestCase( unittest.TestCase ) :
 
+	@staticmethod
+	def waitForDebugger( fn ) :
```

Comment:
Oh, sorry, I was suggesting adding `python/IECore/TestCase.py` and defining this method there, then deriving from that here.

Reply:
Or.. I think I was... I can't find that original comment anymore...

Reply:
It can just be a free function, so shall I jab it into a file called python/IECore/DebugUtil.py?

Reply:
Sure